src/browser/extension/electronMock.js

```diff
@@ -24,7 +24,7 @@ if (
 }
 
 if (window.isElectron) {
-  chrome.storage.local = {
+  chrome.storage.local = chrome.storage.local || {
```

Comment:
Maybe just to wrap in `if (chrome.storage.local) {}` here? So it would be clearer for other contributors what is happening here.